tf_quant_finance/black_scholes/vanilla_prices.py

```diff
@@ -63,12 +66,25 @@ def option_price(volatilities,
     forwards: A real `Tensor` of any shape that broadcasts to the shape of
       `volatilities`. The forwards to maturity. Either this argument or the
       `spots` must be supplied but both must not be supplied.
+    risk_free_rates: An optional real `Tensor` of same dtype as the
```

Comment:
This is very useful and much requested change. Could you then also add continuous dividends as a parameter? 
https://en.wikipedia.org/wiki/Black%E2%80%93Scholes_model#Instruments_paying_continuous_yield_dividends

Reply:
Am I correct in understanding that this would need to be given instead of cost_of_carries? Because both are adjusting similar things just in slightly different ways, (from [wikipedia page](https://en.wikipedia.org/wiki/Cost_of_carry) cost of carries is "the cost of paying dividends when short"), I'm a little suspicious that they are just different ways of monitoring the same thing - to me it would make sense if you could give one or the other. But i could be wrong!

Reply:
You are right, they are treated in the same manner and dividend is incorporated in the `cost_of_carriers`. When `cost_of_carriers` is not provided, it should be set to `risk_free_rates - dividends` in the formula. [Here](https://en.wikipedia.org/wiki/Black%E2%80%93Scholes_model#Instruments_paying_continuous_yield_dividends) is a wikipedia reference.

Please let me know if it makes sence. It should not be much work adding.